lib/plugins/dev-server.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+'use strict';
+
+const pathUtil = require('../config/path-util');
+const AssetOutputDisplayPlugin = require('../friendly-errors/asset-output-display-plugin');
+const FriendlyErrorsPlugin = require('./friendly-errors');
+
+/**
+ * @param {WebpackConfig} webpackConfig
+ * @param {Array} paths to clean
+ * @param {Object} cleanUpOptions
+ * @return {Array} of plugins to add to webpack
+ */
+module.exports = {
+    getPlugins(webpackConfig) {
+        const outputPath = pathUtil.getRelativeOutputPath(webpackConfig);
+        const friendlyErrorsPlugin = FriendlyErrorsPlugin.getPlugins();
+        return [new AssetOutputDisplayPlugin(outputPath, friendlyErrorsPlugin[0])];
```

Comment:
I see your point. Is this issue present in general for all plugins or it is just this particular case with the  `FriendlyErrorsWebpackPlugin`?

Regarding hashed solution:

>... this magic, invented `friendly_errors_plugin` machine name...

I don't like it either. This case looks to me like a good scenario for a **singleton**. I did some research about the topic on the `nodejs` land. I did found some resources:
* [post explaining approaches](https://derickbailey.com/2016/03/09/creating-a-true-singleton-in-node-js-with-es6-symbols/)
* [module implementing a solution](https://www.npmjs.com/package/create-singleton)
* [another implementation approach](https://simplapi.wordpress.com/2012/05/14/node-js-singleton-structure/)

What do you think if we try to implement one of those solutions for that `FriendlyErrorsPlugin`? 

I think this is the best way we can guaranty the single instance we need.